src/main/java/Timo/Timo.java

```diff
@@ -0,0 +1,584 @@
+package Timo;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+import java.util.*;
+import java.io.File;
+import java.time.format.DateTimeFormatter;
+
+class Task {
+    private String taskype = "T";
+    private boolean mark;
+    private String val;
+    public Task(boolean mark, String val) {
+        this.mark = mark;
+        this.val = val;
+    }
+
+    public void markDone() {
+        this.mark = true;
+        return;
+    }
+
+    public void markUndone() {
+        this.mark = false;
+        return;
+    }
+
+    public String getTask() {
+        return this.tasktype;
+    }
+
+    public String getStatusIcon() {
+        return (this.mark ? "X" : " ");
+    }
+
+    @Override
+    public String toString() {
+        return "[" + getStatusIcon() + "] " + this.val;
+    }
+}
+
+class Todo extends Task {
+    private String tasktype = "T";
+
+
+    public Todo(boolean mark, String val) {
+        super(mark, val);
+    }
+
+
+    @Override
+    public String toString() {
+        return "[" + this.tasktype + "]" + super.toString();
+    }
+}
+
+class Deadline extends Task {
+    private LocalDateTime date;
+    private String tasktype = "D";
+
+    public Deadline(boolean mark, String val, LocalDateTime date){
+        super(mark, val);
+        this.date = date;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + this.tasktype + "]" + super.toString() +
+                " (by: " + this.date.format(DateTimeFormatter.ofPattern("MMM dd yyyy HHmm"))
+                + ")";
+    }
+}
+
+class Event extends Task {
+    private String from;
+    private String to;
+    private String tasktype = "E";
```

Comment:
Since tasktype is tied to the Class and does not change, it should be a static variable: `private static String TASKTYPE = "E";`